modules/rtdModule/index.js

```diff
@@ -128,6 +128,31 @@ function getProviderData(adUnits, callback) {
   }
 }
 
+/**
+ * delete invalid data received from provider
+ * this is to ensure working flow for GPT
+ * @param {Object} data received from provider
+ * @return {Object} valid data for GPT targeting
+ */
+function validateProviderDataForGPT(data) {
```

Comment:
Why only validate for GPT?  This module appears only to work with GPT.  What the rational for having an interface that ostensibly supports no string types when GPT doesn't and no other ad server is supported?

Reply:
the module is setting data for GPT and bids,
bids don't have the same limit